build.gradle

```diff
@@ -13,9 +13,8 @@ group = 'io.storj'
 version = '0.7.3'
 
 description = """Java bindings for libstorj"""
-
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
@jkawamoto Is it really necessary to switch to Java 8?

Reply:
java-libstorj is used by android-libstorj. I am worried that this change my cause issues to some android devices. android-libstorj supports API 21+. It's seems that Java 8 is only supported by API 24+.

It should be possible JDK 1.8 to compile with 1.7 compatibility. How do you test this Travis configuration?